add limits to Datatest (#44)
jexia/docs: docs/dataset/README.md

403|Forbidden. Access token does not have permission to insert the record(s) into this dataset.
404|Dataset not found
500|There is an internal error

## Limitations

> Dataset has limit on responds message length - 250Kb. In case you get `response is too big` error try to applay filtering or specify fields in responds.   

Reviewer: ```suggestion
Dataset has a response size limit of **250Kb**. If you receive the error `response is too big`, try to apply filters or fetch only specific fields for the response.
```
